static/app/components/replays/player/scrubber.tsx

```diff
@@ -126,6 +126,37 @@ export const TimelineScrubber = styled(Scrubber)`
   }
 `;
 
+export const NewTimelineScrubber = styled(Scrubber)`
```

Comment:
usually `new` isn't going to be a good long-term name for things. 

How about `SlimTimelineScrubber`?

It's a scrubber still, but I'm realizing that the scrubbing ability might go away once we have the sliding & zooming stuff. Well, maybe not. If not then that'll be an extra iteration/wrinkle to sort out.

Reply:
Ok I will update it! I was thinking of renaming it back to TimelineScrubber once we do feature flag clean up and getting rid of the original TimelineScrubber

Reply:
ya we can rename.. but it's easier to pick a better name now because it'll make the cleanup easier. 

Also there are a few other things with `new`, i'll make suggestions which you an take or improve ontop of.